projects/avro/build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -eu
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+PROJECT=avro
+PROJECT_GROUP_ID=org.apache.avro
+PROJECT_ARTIFACT_ID=avro
+MAIN_REPOSITORY=https://github.com/apache/avro/
+
+MAVEN_ARGS="-Djavac.src.version=15 -Djavac.target.version=15 -DskipTests"
```

Comment:
Java 15 ?! It is not maintained since long time.
Most projects use the Long Time Supported ones - Java 11 or 17. Others use the latest - 20.

Reply:
In OSS-Fuzz the default Java version is 15, we will update it to newer LTS Java version in the future, thank you for your feed back, it is changed to 11 now.